### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by felacek**
Ability to add entire directories of images into database for imls.

### CODE REVIEW

File: plugins/autistic_plugins/Config.js [hunk 1]
```diff
@@ -129,13 +129,19 @@ export default {
...
     },
     addimage: {
-      message: `${p.admin}Succesfully added ${p.highlight}$L[#{url}]this url$L ${p.admin}to the images database.`,
+      message: `${p.admin}Successfully added ${p.highlight}$L[#{url}]this url$L ${p.admin}to the images database.`,
       aliases: ['addimg', 'addimage', 'imlsa'],
       help: `Add an image to the database.`,
       privilege: 1
     },
+    addimages: {
+      message: `${p.admin}Successfully added images from ${p.highlight}$L[#{url}]this url$L ${p.admin}to the images database.`,
```
Comment (wsrvn) [inline plugins/autistic_plugins/Config.js:138]: Well done !

File: plugins/autistic_plugins/FunCommands.ts [hunk 2]
```diff
@@ -135,6 +137,48 @@ tm.commands.add(
...
+      }
+      const xml: Document = new DOMParser({
+        locator: {},
+        errorHandler: {
+          warning: function(w) {},
+          error: function(e) {},
+          fatalError: function (f) {}
+        }
+      }).parseFromString(html, "text/xml")
+      xpath.select(`//a/@href`, xml).map(a => (a as any).value).forEach(a => {
```
Comment (wsrvn) [inline plugins/autistic_plugins/FunCommands.ts:171]: this is sooooooo hot

**Review by wsrvn — APPROVED** [review]
ENORMOUS PR

File: plugins/autistic_plugins/Config.js (hunk 1 above)
Comment (felacek) [reply plugins/autistic_plugins/Config.js:138]: W spelling

**Pull request merged**